ContributedExamples/plot_mht.ipynb

```diff
@@ -4,27 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>Is there a reason you've removed the Atlantic and Pacific portions of this figure? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I could not make them work and since someone else was working on this, I just removed them and was hoping to merge it with their work.

Reply:
I just saw below that you mentioned the other person working on this. I will update the notebook with your suggestions, which are good, and then we can merge with the other notebook.

Reply:
Yeap